Meta.svelte

```diff
@@ -3,15 +3,74 @@
   Metadata helper
 -->
 <script>
-  /** Page title */
+  /** Page title **/
   export let title: Optional<string> = undefined;
-  /** Page description */
+  /** Page description **/
   export let description: Optional<string> = undefined;
-  /** Page image */
+  /** Page image **/
   export let image: Optional<string> = undefined;
+  /** Page url **/
+  export let url: Optional<string> = undefined;
+  /** Site logo **/
+  export let logoUrl: string = url ? `${url}/logo.png` : undefined;
```

Comment:
This seems to opinionated and magic a default, let's just leave as separate to URL. Also since URL can change per page, it would generate bad links on page change. Unless `url` is meant to be root site URL only, in which case the prop name should change

Reply:
Good call, I overlooked the changing URL. Making this `undefined`.